pkg/server/router/router.go

```diff
@@ -54,6 +54,12 @@ const (
 
 	// cloudPathParameterName represents the name of the path parameter for cloud.
 	cloudPathParameterName = "cloud"
+
+	// cloudTypeHeaderName represents the cloud type of the header parameter for cloud.
+	cloudTypeHeaderName = "Cloud-Type"
```

Comment:
This is not needed. The cloud type can be got after get the cloud according to the cloud name.

Reply:
done